pages/blog/july-december-2021-at-asyncapi.md

```diff
@@ -0,0 +1,171 @@
+
+Do you want to be on that list? Watch :point_down: presentation.
+
+<YouTube id="uG_aLF9Z1F0" /> 
+
+There are lots of things yet missing, like TSC list synchronization, voting process automation, and other things that need lots of work, so if you want to contribute to the project and you are a fan of automation, this is the right topic to help with.
+
+## Welcome Studio and Good Bye Playground
+
+We released a new application that you can use to work on your AsyncAPI files. 
+
+Before, you could use [AsyncAPI Playground](https://playground.asyncapi.com/) but soon we'll start redirecting traffic to new, beautiful :point_right: [AsyncAPI Studio](https://studio.asyncapi.com/) :point_left:
+
+The new Studio has a bright future, and you should expect lots of new features coming there. I definitely recommend checking out [Studio's GitHub repository](https://github.com/asyncapi/studio/).
+
+The cool stuff is that it is also integrated with [AsyncAPI CLI](https://www.asyncapi.com/tools/cli):
+```bash
+#install CLI
+npm install -g @asyncapi/cli
+#create new file using an example
+asyncapi new --file-name=asyncapi.yml --example=default-example.yaml --no-tty
+#start studio that picks up asyncapi.yml file from the context it runs in
+asyncapi start studio
+```
+
+Repository with [Playground source code](https://github.com/asyncapi/playground) will most probably be archived and moved to [organization with old archived projects](https://github.com/asyncapi-archived-repos). 
+
+## Hackathon
+
+This year, we hosted our first [AsyncAPI Hackathon](https://www.asyncapi.com/blog/hackathon-faq). Even though we did not have a dedicated marketing campaign, and even though AsyncAPI is not trivial and requires experience to build tools for it, we received 9 submissions.
+
+[Souvik De](https://www.linkedin.com/in/souvik-de-a2b941169) won first prize with [AsyncAPI Bundler](https://github.com/asyncapi/community/discussions/128). Check out Souvik talking about the AsyncAPI Bundler at AsyncAPI Conference.
+
+<YouTube id="j_KLiCRW3t0" /> 
+
+We also had 2 folks that got into the 2nd place:
+- [Ace](https://www.twitter.com/_acebuild) submitted [AsyncAPI Blocks](https://github.com/asyncapi/community/discussions/149)
+- [Greg Meldrum](https://www.linkedin.com/in/greg-meldrum-8b15a03/) submitted [Event Discovery Agent](https://github.com/asyncapi/community/discussions/147)
+
+Congrats to all of you folks!
+
+Check [this discussion to get more details on the voting process](https://github.com/asyncapi/community/discussions/165).
+
+PS. During Hackathon we also had lots of people engaging with us because of Hacktoberfest. We got [30 issues resolved and merged](https://docs.google.com/spreadsheets/d/17YYe-ZompcuUACfCGa8Pc3XunFfWRBrNllIBChJ0yWw/edit#gid=0).
```

Comment:
```suggestion
P.S. During the Hackathon, we also had a lot of people engaging with us because of Hacktoberfest. We got [30 issues resolved and merged](https://docs.google.com/spreadsheets/d/17YYe-ZompcuUACfCGa8Pc3XunFfWRBrNllIBChJ0yWw/edit#gid=0).
```